db/silkworm/db/mdbx.hpp

```diff
@@ -30,6 +30,9 @@
 
 namespace silkworm::db {
 
+constexpr std::string_view kDb_data_file_name{"mdbx.dat"};
```

Comment:
Aren't constants supposed to be CamelCase in Silkworm? e.g. see [tables.hpp](https://github.com/torquem-ch/silkworm/blob/master/db/silkworm/db/tables.hpp)